apps/chat_server/priv/repo/migrations/20171025210428_add_id_to_starred_message_table.exs

```diff
@@ -0,0 +1,9 @@
+defmodule ChatServer.Repo.Migrations.AddIdToStarredMessageTable do
+  use Ecto.Migration
+
+  def change do
+    alter table(:starred_messages) do
+      add :id, :binary_id, primary_key: true
+    end
+  end
+end
```

Comment:
While building out a feature, the model often changes. It's up to you, but it's okay to go back and update the original migration file so there's just one instead of many. Not a blocker!